### DISCUSSION TITLE
Ability to see the list of poll

### DISCUSSION THREAD

**Discussion by Apokalypt** [lukevella/rallly#333] [Ideas]
Hi,

I think it would be interesting to be able to see the list of polls currently running and those that are locked via a list. It would be easier to consult the result of the polls when there are several of them.
Also, it would allow to see some information at a glance without having to click on it

**Comment by lukevella** [comment]
I hear you, it's in the works and will be a part of a major update to the app. 👍

**Reply by Apokalypt** [reply]
Interesting, if you need any help to work on the project i would be really happy to help you

**Reply by lukevella** [reply]
Thanks @Apokalypt. I think there will be opportunities to contribute in the future but I'm still currently laying the foundation for now.

**Reply by Apokalypt** [reply]
Alright, if ever there is a need do not hesitate to contact me and if you are concerned about my skills I can reassure you that this is currently my job (software development) so there will be no worries
